src/_includes/services_page_intro.liquid

```diff
@@ -1,5 +1,40 @@
 <header>
-  <div class="services-page-intro__image"></div>
+  <div class="services-page-intro__image">
+    <div class="services-page-intro__navigation">
+      <div class=services-page-intro__icon-container>
+        <a href="{{'/services/software-modernisation/' | prepend: site.baseurl }}" class="services-page-intro__navigation-link">
+          <div class="services-page-intro__navigation-icon services-page-intro__icon--software-modernisation">
+            {{ site.data.service_lines.software-modernisation.icon }}
+          </div>
+          <h2 class="services-page-intro__navigation-text">{{site.data.service_lines.software-modernisation.name}}</h2>
+        </a>
+      </div>
+      <div class=services-page-intro__icon-container>
+        <a href="{{'/services/software-delivery/' | prepend: site.baseurl }}" class="services-page-intro__navigation-link">
+          <div class="services-page-intro__navigation-icon services-page-intro__icon--software-delivery">
+            {{ site.data.service_lines.software-delivery.icon }}
+          </div>
+          <h2 class="services-page-intro__navigation-text">{{site.data.service_lines.software-delivery.name}}</h2>
+        </a>
+      </div>
+      <div class=services-page-intro__icon-container>
```

Comment:
There is a lot of repition here. Maybe this could be extracted to an include at some point and passed the data it needs. 

It may even be possible to set up a for loop to loop through each entry within the data file and output the includes. It wouldn't need to be a blocker if we want to get this live now.